handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -256,6 +256,13 @@ public static boolean isAlpnSupported() {
     }
 
     /**
+     * Returns {@code true} if the used version of OpenSSL supports OCSP stapling.
+     */
+    public static boolean isOcspSupported() {
+      return version() >= 0x10002000L;
```

Comment:
In case this is where the confusion comes from, BoringSSL's APIs has *no relation* to `OPENSSL_NO_OCSP`. They are available regardless.

We do not, in general, have individual toggles for all our features like OpenSSL does. Instead, we define a set of `OPENSSL_NO_*` values based on what makes portability simplest. These symbols do not affect what features BoringSSL exposes.